config/initializers/apipie.rb

```diff
@@ -5,7 +5,6 @@
   config.app_name = 'Foreman'
   config.app_info = 'The Foreman is aimed to be a single address for all machines life cycle management.'
   config.doc_base_url = '/templates_doc'
-  config.markup = ApipieDSL::Markup::Markdown.new if Rails.env.development? && defined? Maruku
```

Comment:
why is this required?

Reply:
AFAIK we don't ship `maruku` gem, so there is no point to use `Mardown` markup for descriptions in dev mode since they will be broken in prod.